{
  try {
    testable.test();
    fail(String.format("Expected exception, %s, but not thrown",clazz));
  }
 catch (  Exception e) {
    if (!clazz.isInstance(e)) {
      fail(String.format("Expected exception of %s but got %s",clazz,e.getClass()));
    }
  }
}
